Dockerfile

```diff
@@ -145,6 +145,7 @@ RUN apk --update --no-cache add \
     php7-zip \
     php7-zlib \
     python3 \
+    py3-pip \
```

Comment:
Why `pip` is required? Do you have a link to the plugin?

Reply:
I'm not sure actually, python is not my main language.
But without pip it gave warnings about not finding the `requests` package.

The rutorrent cloudflare plugin uses [cloudscraper](https://github.com/VeNoMouS/cloudscraper) that uses requests. 
I just found the solution by reading the rutorrent [wiki page](https://github.com/Novik/ruTorrent/wiki/Plugin_cloudflare).

There might be a better solution but this seems to work

Reply:
Looking at the documentation, `pip` is not required so you can revert this change here.

Reply:
Tried reverting this and got 
"`[2020-09-22 13:55:32] _cloudflare plugin: cloudscraper module can't be loaded in Python`" in the log. 
So `pip` seems required despite what the docs says unfortunately.

Reply:
I have made some changes to [install clouscraper through pip3](https://github.com/crazy-max/docker-rtorrent-rutorrent/commit/fd59f739f49d7307cd29d52739b7ad95a52d17f6). And looking [at clouscraper detection on plugin code](https://github.com/Novik/ruTorrent/blob/master/plugins/_cloudflare/cloudflare.php#L27) it should be safe to not include pip.